components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -281,13 +281,23 @@ - (void)layoutSubviews {
   CGSize leadingButtonBarSize = [_leadingButtonBar sizeThatFits:self.bounds.size];
   CGRect leadingButtonBarFrame =
       CGRectMake(0, CGRectGetMinY(self.bounds), leadingButtonBarSize.width, leadingButtonBarSize.height);
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    leadingButtonBarFrame.origin.x = self.safeAreaInsets.left;
```

Comment:
Do we want to set it equal to safeAreaInsets or += safeAreaInsets?

Reply:
leadingButtonBarFrame.origin.x is 0 as per the line above. I can change it to += if you want, but that isn't necessarily a better approach imo.